projects/dsp-ui/src/assets/style/_viewer.scss

```diff
@@ -160,3 +160,30 @@ button.save .mat-icon,
 button.cancel .mat-icon {
   font-size: 18px;
 }
+
+.deletion-dialog .title {
```

Comment:
Is this more general than value deletion? Wouldn't it make more sense to keep it in the component's CSS file?

Reply:
After some thinking, I should refactor this a little. The current ConfirmationMessage component should be renamed and made more specific to a deletion message. This way the ConfirmationDialog component can be reused and you can just supply it with the Message component of your choosing.

Ideally the ConfirmationDialog component is just a shell with a button to confirm and a button to cancel and its contents is defined by the type of message component you give it. We should handle this in another PR though.